### ISSUE TITLE
The training code

### ISSUE THREAD

**Issue by Jarvx** [facebookresearch/swe-rl#9]
Hi, 

Thanks for sharing this interesting work. I was wonder if you'll share the code for training your model?

thanks

**Comment by UniverseFly** [comment]
We currently don't have the plan to release the training code, but as discussed in the paper, our approach is designed to be general where we simply adopted GRPO with the SWE-RL reward function defined [here](https://github.com/facebookresearch/swe-rl/blob/e4f9dd033d25d628dc88543312cb88aadd03cf44/src/swerl/core/reward.py#L221-L260). It can be directly applied to open-source RL libraries with GRPO support (e.g., [verl](https://github.com/volcengine/verl)).
